src/sas/qtgui/Calculators/GenericScatteringCalculator.py

```diff
@@ -534,7 +560,26 @@ def change_data_type(self):
         self.update_gui()
         self.check_for_magnetic_controls()
 
+    def change_qValidator(self):
+            if float(self.txtQxMax.text()) == 0:
```

Comment:
I spoke to @yunliu01 about this and he seems to think the values you have here are acceptable, but this is a placeholder for future work using Rg to estimate the Q-range that should be used. I would suggest you add a  comment describing that future work otherwise this coercion of Q values looks suspect.

Reply:
Has this been resolved?

Reply:
A comment has been added to this part of the code.